containers/Connector/Connector.tsx

```diff
@@ -15,6 +15,8 @@ import {
 	synthetix,
 } from '@synthetixio/contracts-interface';
 import { ethers } from 'ethers';
+// @ts-ignore
+import snx from 'synthetix';
```

Comment:
I would try to pull everything from [@synthetixio/contracts-interface](https://github.com/Synthetixio/js-monorepo/tree/master/packages/contracts-interface) if possible. In this case we may have to expose new functionality.